src/Hydra/Serializer/ErrorNormalizer.php

```diff
@@ -42,6 +43,10 @@ public function normalize($object, $format = null, array $context = [])
         $message = $object->getMessage();
         if ($this->debug) {
             $trace = $object->getTrace();
+        } else {
```

Comment:
We need to check `$object instanceof FlattenException` first. And I think it's a bad idea to use `<=` for this.